src/System.Private.CoreLib/shared/System/Range.cs

```diff
@@ -95,15 +95,15 @@ public override string ToString()
         /// <summary>Create a Range object starting from first element to the end.</summary>
         public static Range All => new Range(Index.Start, Index.End);
 
-        /// <summary>Destruct the range object according to a collection length and return the start offset from the beginning and the length of this range.</summary>
-        /// <param name="length">The length of the collection that the range will be used with. length has to be a positive value</param>
+        /// <summary>Calculate the start offset and length of range object using a collection length.</summary>
+        /// <param name="length">The length of the collection that the range will be used with. length has to be a positive value.</param>
         /// <remarks>
         /// For performance reason, we don't validate the input length parameter against negative values.
         /// It is expected Range will be used with collections which always have non negative length/count.
         /// We validate the range is inside the length scope though.
         /// </remarks>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public OffsetAndLength GetOffsetAndLength(int length)
+        public (int offset, int rangeLength) GetOffsetAndLength(int length)
```

Comment:
The return type in https://github.com/dotnet/corefx/issues/35508 is `(int Offset, int Length)`. Is it intentional that this is different?

Reply:
It should be `int Offset, int Length`

Reply:
I'll fix that.